src/charm.py

```diff
@@ -272,7 +258,7 @@ def _pebble_layer(self) -> Layer:
                 SERVICE_NAME: {
                     "override": "merge",
                     "summary": "OpenFGA",
-                    "command": f"sh -c 'openfga run --log-format json --log-level {self._log_level} 2>&1 | tee -a {LOG_FILE_PATH}'",
+                    "command": f"sh -c 'openfga run --log-format json --log-level {self._log_level} 2>&1'",
```

Comment:
Do we need to keep writing `stderr` to `stdout`(`2>&1`)?  Maybe pebble handles this on it's own?